examples/siro_sandbox/hydra_helper.py

```diff
@@ -13,7 +17,45 @@
 )
 
 
+class HabitatHitlConfigPlugin(SearchPathPlugin):
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
```

Comment:
This is required so that e.g. when Hydra parses `basic_viewer.yaml` and sees `hitl_defaults.yaml` in the `defaults` section, it can find that file.